packages/types/src/appearance.ts

```diff
@@ -426,79 +429,72 @@ export type Variables = {
   colorNeutral?: CssColorOrAlphaScale;
   /**
    * The default text color.
-   * @default #131316
+   * @default '#212126'
    */
   colorText?: CssColor;
   /**
    * The text color for elements of lower importance, eg: a subtitle text.
-   * @default A lighter shade of {@link Variables.colorText}
```

Comment:
I think we need to keep this comment. The default changes if `colorText` changes.

Reply:
I think we should explicitly say the default here, and maybe move this comment up to the descriotion